src/shared/copy/notifications.ts

```diff
@@ -1345,7 +1345,7 @@ export const inviteFailed = (): Notification => ({
 
 export const invitationResentSuccessful = (): Notification => ({
   ...defaultSuccessNotification,
-  message: `Invitation Sent`,
+  message: `Invitation resent`,
```

Comment:
I think it should be `re-sent`. `resent` has a different meaning 😅

Reply:
ya, hilarious I saw that and thought, something doesn't look right. Will do